hushline/routes/forms.py

```diff
@@ -62,6 +63,15 @@ class F(FlaskForm):
 
         self.F = F
 
+        # Add email body hidden field
+        setattr(
```

Comment:
You can just do this like a normal `FlaskForm`  in the class definition for `F`. Not with `setattr`

Reply:
Good point